src/web/components/Pagination/Pagination.test.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Pagination from './';
+import Button from '../Button';
+
+describe('Pagination', () => {
```

Comment:
It seems that some limit test are missing:

- What if I'm on page 1 and that I try to click `select page - 1`
- What if I'm on a `maxPage` (like 100) that I try to click on `select page + 1`

😊